lms/djangoapps/instructor/management/commands/dump_userinfo.py

```diff
@@ -259,10 +264,16 @@ def handle(self, *args, **options):
 
             certificate = self.add_fields_to(student_dict, CERTIFICATE_FIELDS, certificate_table, user_id)
 
-            # If the user has received a certificate, adjust their credits issued and course completion flag.
+            # If the student has hit the 'Grade Me' button.
             if student_dict['Credit Date']:
+                student_dict['Attested'] = True
+
+            # If the student has received a certificate, adjust their credits issued and certificate flag.
+            if student_dict['Certificate'] in CERTIFICATE_STATUSES:
                 student_dict['Credits Issued'] = num_credits
-                student_dict['Certif'] = True
+                student_dict['Certificate'] = True
+            else:
+                student_dict['Certificate'] = False
```

Comment:
If you rename the default when student_dict is defined you ca remove this else clause

Reply:
This has to be set here, cause we're checking the very field that's being set here...